### ISSUE TITLE
A11y issue - multiselect focus not following WAI standard

### ISSUE THREAD

**Issue by jakub-wisniowski-wrx** [ng-select/ng-select#2736]
### Describe the bug

The standard for multi-select listbox focus is described [here](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/) in section 'Keyboard interaction'. It says:

```
When a multi-select listbox receives focus:
- If none of the options are selected before the listbox receives focus, focus is set on the first option and there is no automatic change in the selection state.
- If one or more options are selected before the listbox receives focus, focus is set on the first option in the list that is selected.
```

### Steps to Reproduce

- Select two values in multiselect
- focus ng-select
- click arrow down to show the listbox

Last instead of first selected element on the list is focused.

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

https://ng-select.github.io/ng-select/#/multiselect

### Expected behaviour

First selected element on the list should be focused

### What version of the library you see this issue?

21.0.0, 14.9.0

### What version of Angular are you using?

Angular 19

### Desktop? Please complete the following information

OS: Ubuntu 24
Browser: Brave 1.85.111, Chromium 142.0.7444.175, Firefox 145.0.2
